README.md

```diff
@@ -43,11 +43,14 @@ Then you'll just need to add the messages template with $CLASS_NAME_message.txt.
 
 ### Adding A New Event
 
-To create a new event simply define a new method with the same name as the method you want to handle.
+To create a new event, within the [operationcode_bot.rb](https://github.com/OperationCode/operationcode_bot/blob/master/operationcode_bot.rb) file, simply define a new `private` method with the same name as the method you want to handle.
+
 For example, if you wanted to write some code any time someone [pins an item](https://api.slack.com/events/pin_added) (the ```pin_added``` event) you would define
```

Comment:
Thanks for pointing this out @hpjaj, this is good to know (that a new private method needed to be added to the [operationcode_bot.rb](https://github.com/OperationCode/operationcode_bot/blob/master/operationcode_bot.rb) file.  I missed this step in PR #87, will add.  I thought that somehow a new class instance (matching the event name) would be automatically instantiated if it somehow matched the event name from the Slack Events API.